wp-content/civi-extensions/goonjcustom/Civi/InstitutionService.php

```diff
@@ -139,35 +139,40 @@ private static function addContactToGroup($contactId, $groupId) {
    * @param object $objectRef
    *   The parameters that were sent into the calling function.
    */
-  public static function setOfficeDetails($op, $groupID, $entityID, &$params) {
+  public static function organizationCreated(string $op, string $objectName, int $objectId, &$objectRef) {
 
-    if ($op !== 'create' || self::getOrgSubtypeName($entityID) !== self::ENTITY_SUBTYPE_NAME) {
-      return;
+    if ($op !== 'create' || $objectName !== 'Organization') {
+      return FALSE;
     }
 
-    if (!($stateField = self::findStateField($params))) {
-      return;
+    $subTypes = $objectRef->contact_sub_type;
+
+    if (empty($subTypes)) {
```

Comment:
```suggestion
    if (!$subTypes) {
```

Reply:
But both of these things are the same. Why do we need to make a change like this?